### PR TITLE
Apply abandoned API documentation correction #5992 

### PR DESCRIPTION
**Pull request by zafar-hussain**
#### Summary

API documentation correction, closes #5992

#### Ticket Link
https://github.com/mattermost/docs/issues/5992

### CODE REVIEW

**Comment by zafar-hussain** [comment]
hi @cwarnermm ,

Please check and advice on my PR, related to issue https://github.com/mattermost/docs/issues/5992, "Help Wanted: Apply abandoned API documentation correction"

If there is any more work to be done, please let me know.

thanks

File: v4/source/commands.yaml [hunk 1]
```diff
@@ -492,7 +492,8 @@
                   description: Channel Id where the command will execute
                 command:
                   type: string
-                  description: The slash command to execute
+                  ##description: The slash command to execute
```
Comment (cwarnermm) [inline v4/source/commands.yaml:495]: Please remove this line

**Review by cwarnermm — COMMENTED** [review]
Thank you, @zafar-hussain! I've provided feedback inline.

File: v4/source/commands.yaml (hunk 1 above)
Comment (zafar-hussain) [reply v4/source/commands.yaml:495]: line removed

**Review by cwarnermm — APPROVED** [review]
Thank you, @zafar-hussain!

**Pull request merged**